CMakeLists.txt

```diff
@@ -17,6 +17,9 @@ option(WITH_TEST "build with test" OFF)
 option(WITH_COVERAGE "build with coverage" OFF)
 option(WITH_DOC "build with documenation" OFF)
 option(WITH_VCHAN "build with vchan" ON)
+option(WITH_IAM_API "build core api iam lib" ON)
```

Comment:
Set external project option: It should not be option but rather we should set option for submodule. See SM external project options:
https://github.com/aosedge/aos_core_sm_cpp/blob/775fb6bdc302ef9398c5be73f2a7324e27e3c044/CMakeLists.txt#L29

Reply:
Done